blog/_posts/2022-03-02-build-a-resource-catalog.md

```diff
@@ -0,0 +1,86 @@
+
+```shell
+# Start Tilt with one resource
+$ tilt up storage
+
+# Oh wait, let's add these other resources I need to work on
+$ tilt args storage frontend glitch object-detector color
+
+# Ahh, my work changed and I need a different set running
+$ tilt args storage frontend object-dectector bounding-box muxer
+```
+
+Now, Tilt makes it easier to enable and disable resources (and sets of resources!). You don't need to restart Tilt or copy and paste long [`tilt args` commands][tilt-args] when you want to work with different resources.
+
+In your Tiltfile, you can define the resources that automatically run on `tilt up,` and all other resources will be disabled, but available to enable through Tilt’s UI and CLI. The Tilt UI lets you show or hide disabled services. And in the table view, you can select multiple resources to enable or disable at once. When you’re viewing logs, there’s also a handy way to disable troublesome resources. [See our docs for a more detailed walkthrough.][disable-resources]
+
+![disable a single resources](/assets/images/disable-resources/single-resource-disable.gif)
+
+## If you’re setting up Tilt for your team...
+
+Enabling and disabling resources on the fly should make workflows easier for individual contributors focused on shipping. But, let’s zoom out and talk about how the idea of a “resource catalog” is a way to use Tilt to improve the overall development experience for your team.
+
+```python
+# In your Tiltfile
+config.define_string_list("to-run", args=True)
+cfg = config.parse()
+
+groups_by_team = {
+  'client_team': ['a', 'b', 'c', 'd', 'm', 'p'],
+  'data_team': ['b', 'd', 'm', 'n']
+  'infra_team': ['x', 'y', 'z']
+}
+
+group_to_run = cfg['to-run']
+if group_to_run:
+  config.set_enabled_resources(groups_by_team[group_to_run])
+```
+
+You can define sensible defaults for engineers, like grouping services based on those frequently used by particular teams, or based on your stack. Engineers can start Tilt with these [predefined groups][tiltfile-config], but they won’t be excessively constrained by them. Both Tilt’s UI and CLI makes it easy to override the running set of services and add or subtract based on what they need.
+
+
+Organizing your resources with labels is particularly useful because engineers can quickly enable and disable using these groups in the UI.
+
+![enable multiple resources](/assets/images/disable-resources/bulk-disabling.gif)
+
+One caveat to note: enabling and disabling individual resources doesn’t take into account any dependencies they rely on. (Though we have several [feature requests][disable-issues] here!) If you’ve got resources with a lot of dependencies, start with the Tiltfile config and use `tilt args` to define sets, since it will take dependencies into account.
```

Comment:
i'm not sure this caveat / note needs to be here.

Reply:
I definitely think it's useful to add this